src/data/gps.js

```diff
@@ -0,0 +1,930 @@
+        },
+        {
+          name: 'Oran',
+          gps: [35.6969, -0.6331],
+        },
+        {
+          name: 'Constantine',
+          gps: [36.365, 6.6147],
+        },
+        {
+          name: 'Annaba',
+          gps: [36.907, 7.7411],
+        },
+        {
+          name: 'Batna',
+          gps: [35.5557, 6.1741],
+        },
+      ],
+    },
+    {
+      name: 'Kenya',
+      continent: 'Africa',
+      gps: [-1.2864, 36.8172],
+      map: Kenya,
+      cities: [
+        {
+          name: 'Nairobi',
+          gps: [-1.2864, 36.8172],
+        },
+        {
+          name: 'Mombasa',
+          gps: [-4.0435, 39.6682],
+        },
+        {
+          name: 'Kisumu',
+          gps: [-0.0917, 34.7676],
+        },
+        {
+          name: 'Nakuru',
+          gps: [-0.3031, 36.08],
+        },
+        {
+          name: 'Eldoret',
+          gps: [0.5204, 35.2699],
+        },
+      ],
+    },
+    {
+      name: 'Ethiopia',
+      continent: 'Africa',
+      gps: [9.145, 40.4897],
+      map: Ethiopia,
+      cities: [
+        {
+          name: 'Addis Ababa',
+          gps: [9.145, 38.74],
+        },
+        {
+          name: 'Dire Dawa',
+          gps: [9.59, 41.87],
+        },
+        {
+          name: 'Mekelle',
+          gps: [13.4967, 39.475],
+        },
+        {
+          name: 'Gondar',
+          gps: [12.609, 37.464],
+        },
+        {
+          name: 'Hawassa',
+          gps: [7.056, 38.477],
+        },
+      ],
+    },
+    {
+      name: 'China',
+      continent: 'Asia',
+      gps: [35.8617, 104.1954],
+      map: China,
+      cities: [
+        {
+          name: 'Shanghai',
+          gps: [31.2304, 121.4737],
+        },
+        {
+          name: 'Beijing',
+          gps: [39.9042, 116.4074],
+        },
+        {
+          name: 'Guangzhou',
+          gps: [23.1291, 113.2644],
+        },
+        {
+          name: 'Shenzhen',
+          gps: [22.5431, 114.0579],
+        },
+        {
+          name: 'Chengdu',
+          gps: [30.5728, 104.0668],
+        },
+      ],
+    },
+    {
+      name: 'India',
+      continent: 'Asia',
+      gps: [20.5937, 78.9629],
+      map: India,
+      cities: [
+        {
+          name: 'Mumbai',
+          gps: [19.076, 72.8777],
+        },
+        {
+          name: 'Delhi',
+          gps: [28.7041, 77.1025],
+        },
+        {
+          name: 'Bangalore',
+          gps: [12.9716, 77.5946],
+        },
+        {
+          name: 'Kolkata',
+          gps: [22.5726, 88.3639],
+        },
+        {
+          name: 'Chennai',
+          gps: [13.0827, 80.2707],
+        },
+      ],
+    },
+    {
+      name: 'Japan',
+      continent: 'Asia',
+      gps: [36.2048, 138.2529],
+      map: Japan,
+      cities: [
+        {
+          name: 'Tokyo',
+          gps: [35.6895, 139.6917],
+        },
+        {
+          name: 'Yokohama',
+          gps: [35.4437, 139.638],
+        },
```

Comment:
- [ ] In alignment with the previous [review](https://github.com/wacoo/global-air-polution-tracker/pull/5#discussion_r1326860815) feedback, kindly ensure that you modify your project to fetch data from the Redux store via API calls, as opposed to manual data entry, to guarantee the presence of real-time, up-to-date information and maintain the highest professional coding standards. :wink:

Reply:
I tried that. I cant find a free api. I found either limited requests or not free.

Reply:
For the countries and cities I mean.

Reply:
Great work replacing the hard-coded country data with an api. Good work using redux to call the api